packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -715,10 +715,6 @@ describe('plugin-meetings', () => {
           assert.exists(meeting.setMeetingQuality);
         });
 
-        it('should set mediaProperty with the proper level', () => meeting.setMeetingQuality(CONSTANTS.QUALITY_LEVELS.LOW).then(() => {
```

Comment:
`setMeetingsQuality` doesn't change the `mediaProperties` directly anymore so this test case will always fail since `setLocalQualityLevel` and `setRemoteQualityLevel` are stubbed and they are handling the property change